llvm/tools/repo2obj/repo2obj.cpp

```diff
@@ -1,9 +1,8 @@
-//===- repo2obj - Convert a Repository ticket to an ELF object file ------===//
+//===- repo2obj.cpp - Convert a Repository ticket to an ELF object file ---===//
```

Comment:
A second dash? (Since we're changing the text anyway.)

Reply:
> Should we give one dash or two dashes?

I don't think it matters!

>>Should this include the path? (Since we're changing the text anyway.)
>No, I checked other files (our tools and other LLVM tools). Only file name is provided, not include the path.

The file elf.cmake includes the path. There’s some inconsistency.

Reply:
Thanks Paul.

To be consistent with other part of Repo codes, I have updated patch using one dash and removed the paths in the elf.cmake and repo.cmake.